src/locutus/__init__.py

```diff
@@ -13,19 +13,31 @@ def strip_none(value):
     return value
 
 
-def fix_varname(varname):
-    return varname.strip().replace(" ", "_")
-
-
-def clean_varname(name):
-    return (
-        name.lower()
-        .replace(" ", "_")
-        .replace("(", "")
-        .replace(")", "")
-        .replace("'", "")
-        .replace('"', "")
-    )
+# Special character mappings. UTF-8 Hex
+sp_char_mappings = {'/': '0x2F', 
+                    '.': '0x2E' # Note: '..' will become '0x2E0x2E'
+                    }
+
+def get_code_index(code):

```

Comment:
If they keep finding bugs that you have to fix, we'll finally get our code commented! :)